x-pack/plugins/siem/server/graphql/ip_details/resolvers.ts

```diff
@@ -54,13 +61,22 @@ export const createIpDetailsResolvers = (
       return libs.ipDetails.getDomains(req, options);
     },
     async DomainFirstLastSeen(source, args, { req }) {
-      const options = {
+      const options: DomainFirstLastSeenRequestOptions = {
```

Comment:
👍 on Types

Reply:
Thanks for pointing these out! Happy with all the type-cleanup and imrpovements we've been doing lately :D